streams/src/main/java/org/apache/kafka/streams/state/internals/CachingWindowStore.java

```diff
@@ -179,6 +182,7 @@ public synchronized void put(final Bytes key, final byte[] value, final long win
     }
 
     @Override
+    @Deprecated
```

Comment:
These don't need to be marked deprecated, thanks to the non-deprecated definition in `WindowStore` now.

This comment applies to all the implementations of `WindowStore` in this PR.